kratos/python_scripts/vtk_output_process.py

```diff
@@ -0,0 +1,57 @@
+import KratosMultiphysics
+import os
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return VtkOutputProcessPython(Model, settings)
+
+
+## All the processes python should be derived from "Process"
+class VtkOutputProcessPython(KratosMultiphysics.Process):
+    def __init__(self, Model, settings ):
+        KratosMultiphysics.Process.__init__(self)
+
+        default_parameters = KratosMultiphysics.Parameters("""
+        {
+            "model_part_name"                    : "PLEASE_SPECIFY_MOEL_PART_NAME",
```

Comment:
The ideal would be to be the closest possible to the gid_process

Reply:
This is the closest possible I could get.

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/50024760-b5f21380-ffe3-11e8-93f2-e11a586fd544.png)

I think that except "gidpost_flags" which is very specific, the rest can be copied. We can do a similar thing like "vtkpost_flags" with the equivalent configurations. Even for GP results (we can implement it later)

Reply:
He is using the same configuration as the hdf output, which I also prefer

Reply:
Anyway we should unify this in the near future

Reply:
Shall we discuss in a separate issue already ?